### DISCUSSION TITLE
搬出東京就給你一百萬！日本新政策為何不吸引人

### DISCUSSION THREAD

**Discussion by HuangHsinTzu** [1101-GS4719-C/Python-Community-GS4719-C#1048] [General]
鼓勵搬離東京補助計畫 根據日本經濟新聞報導，2019 年已啟動一項補助計畫，旨在鼓勵有孩子的家庭搬到「少子化和老齡化人口地區」
但其實很多日本網友認為幾乎每有人就因為這種小事就決定搬家
事實上，如果考慮為什麼年輕人會從鄉村流向東京，就會發現這不是一個可以用一次性補貼來解決的問題
台灣其實鄉村人口外流的情形其實也挺嚴重的
我自己是覺得要阻止鄉村地區的少子化及老齡化問題真的太難了
老年人住不習慣都市，年輕人口又因為估作機會太少不願意回到鄉村
所以更實際的做法是加強鄉村地區的福利制度和長照制度
https://tw.news.yahoo.com/%E6%90%AC%E5%87%BA%E6%9D%B1%E4%BA%AC%E5%B0%B1%E7%B5%A6%E4%BD%A0-%E7%99%BE%E8%90%AC-%E6%97%A5%E6%9C%AC%E6%96%B0%E6%94%BF%E7%AD%96%E7%82%BA%E4%BD%95%E4%B8%8D%E5%90%B8%E5%BC%95%E4%BA%BA-065712440.html

**Comment by Valentinaa0214** [comment]
雖然這樣能夠減輕東京的人口壓力和交通壓力、促進其他地區的發展、減少居民對於房價等的困擾。但這個政策同時可也可能會對東京的經濟和文化造成傷害，以及造成其他地區的房價上漲。解決東京人口過多問題的方法，需要在促進其他地區發展和減少東京人口壓力之間取得平衡，並考慮到所有相關的經濟、社會和文化影響，是一個頗困難且複雜的問題。
